src/equal.c

```diff
@@ -432,8 +437,8 @@ bool equal_names(SEXP x, SEXP y) {
 
 static SEXP df_equal_na(SEXP x, R_len_t size);
 
-// [[ register() ]]
-SEXP vctrs_equal_na(SEXP x) {
+// [[ include("vctrs.h") ]]
```

Comment:
Are we now avoiding the main header?

```suggestion
// [[ include("equal.h") ]]
```